pkg/apis/operator.min.io/v1/helper.go

```diff
@@ -415,6 +415,22 @@ func (mi *MinIOInstance) CreateMCSUser(minioSecret, mcsSecret map[string][]byte)
 	return nil
 }
 
+// Validate returns an error if any configuration of the MinIO instance is invalid
+func (mi *MinIOInstance) Validate() error {
+	// Make sure the storage request is not 0
+	if mi.Spec.VolumeClaimTemplate.Spec.Resources.Requests.Storage().Value() == 0 {
```

Comment:
should we validate storage value is not less than 1 (negative numbers)?

Reply:
yes good idea @Alevsk